### PR TITLE
Add realestate.com.au

### PR DESCRIPTION
**Pull request by fletcherdodd**

### CODE REVIEW

File: entries/r/realestate.com.au.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "realestate.com.au": {
+    "domain": "realestate.com.au",
+    "contact": {
+      "facebook": "realestateAus",
+      "twitter": "realestate_au"
+    },
+    "categories": [
+      "investing"
+    ]
```
Comment (hkamran80) [inline entries/r/realestate.com.au.json:17]:
```suggestion
    ],
    "regions": [
      "au"
    ]
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]
Do you know if their other sites (REA Group, realcommercial.com.au, PropTrack, etc.) use the same authentication backend?

**Comment by fletcherdodd** [comment]
> Do you know if their other sites (REA Group, realcommercial.com.au, PropTrack, etc.) use the same authentication backend?

Just realcommercial.com.au & property.com.au do. They direct you to id.realestate.com.au to authenicate. Should I just add the others as additional domains. Or adjust the name also?

**Comment by hkamran80** [comment]
> > Do you know if their other sites (REA Group, realcommercial.com.au, PropTrack, etc.) use the same authentication backend?
> 
> Just realcommercial.com.au & property.com.au do. They direct you to id.realestate.com.au to authenicate. Should I just add the others as additional domains. Or adjust the name also?

Adding them as additional domains should suffice.

**Review by Carlgo11 — APPROVED** [review]

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**